### PR TITLE
add a `query` command

### PR DESCRIPTION
**Pull request by gotcha**
for continuous integration

I am pretty happy with the code. I have some doubt about what to do in case of multiline values.

Feedback appreciated.

### CODE REVIEW

**Comment by gotcha** [comment]
@mgedmin Thanks for the review... and reminding me the changelog.

**Comment by gotcha** [comment]
@reinout Thanks for the review.

File: src/zc/buildout/buildout.py [hunk 1]
```diff
@@ -2115,7 +2140,7 @@ def main(args=None):
         command = args.pop(0)
         if command not in (
             'install', 'bootstrap', 'runsetup', 'setup', 'init',
-            'annotate',
+            'annotate', 'query',
```
Comment (leorochael) [inline src/zc/buildout/buildout.py]: I wish there was a way not to have to repeat command names here.

Perhaps a simple `@command` decorator that appends method names to a list of commands...

Anyway, just a thought, not an actual request for change.

Comment (gotcha) [reply src/zc/buildout/buildout.py]: Nice idea !

**Comment by gotcha** [comment]
@leorochael Thanks for the review.

**Comment by gotcha** [comment]
@leorochael Would you review the last commit ? following your suggestion...

**Comment by leorochael** [comment]
> @leorochael Would you review the last commit ? following your suggestion...

It's perfect!

**Comment by arterrey** [comment]
Can I suggest returning a non-zero status after handling the KeyError?

**Comment by gotcha** [comment]
@arterrey Good point !

**Comment by gotcha** [comment]
released at https://pypi.org/project/zc.buildout/2.13.0/

**Pull request merged**